src/components/applications/UserDashboard.tsx

```diff
@@ -0,0 +1,32 @@
+"use client";
+import * as React from "react";
+import CssBaseline from "@mui/material/CssBaseline";
+import { Box } from "@mui/material";
+import { useMUITheme } from "@/hooks/theme";
+import { ThemeProvider as MUIThemeProvider } from "@mui/material/styles";
+import { useOidcAccessToken } from "@axa-fr/react-oidc";
+
+/**
+ * Build the User Dashboard page
+ * @returns User Dashboard content
+ */
+export default function UserDashboard() {
+  const theme = useMUITheme();
```

Comment:
Does this autodetect the client's system wide preference? (o.e. `window.matchMedia('(prefers-color-scheme: dark)')`)

Reply:
Good point, it should autodetect the client's system wide preference now